src/main/java/seedu/duke/tasks/Task.java

```diff
@@ -3,17 +3,21 @@
 public class Task {
     public static final String ICON_UNCOMPLETED = "( )";
     public static final String ICON_COMPLETED = "(X)";
-    public static final String TASK_STRING_NO_DESC = "%s %s";
-    public static final String TASK_STRING_WITH_DESC = "%s %s (%s)";
+    public static final String TASK_STRING_NO_DESC_NO_TIME = "%s %s";
+    public static final String TASK_STRING_WITH_DESC_NO_TIME = "%s %s (%s)";
+    public static final String TASK_STRING_NO_DESC_WITH_TIME = "%s %s (Estimated Working Time: %s)";
+    public static final String TASK_STRING_WITH_DESC_WITH_TIME = "%s %s (%s) (Estimated Working Time: %s)";
 
     private boolean isTaskDone;
     private String taskName;
     private String taskDescription;
+    private String estimatedWorkingTime;
 
-    public Task(String taskName, String taskDescription) {
+    public Task(String taskName, String taskDescription, String estimatedWorkingTime) {
         this.taskName = taskName;
         this.taskDescription = taskDescription;
         this.isTaskDone = false;
+        this.estimatedWorkingTime = estimatedWorkingTime;
     }
 
```

Comment:
Suggest to add overload constructor here

Reply:
i think adding constructors here might be infeasible giventhat both description and working time are strings. Perhaps we can change it if we decide to change the time to another data type

Reply:
I understood the concern about discussing with Hee Kit. Because both description and working time are optional here, and currently both are string, a construction in form of Task(String taskName, String parameter2) cannot differentiate the whether parameter2 refers to taskDescription or estimatedWorkingTime. We decide to keep it unchanged temporarily and make an issue about the order constraint of "-t" and "-d". Later when we make a distinct class for the time, we can apply overloading constructor to differentiate this.